config.yaml

```diff
@@ -97,7 +97,7 @@ languages:
             - text: Citing SciPy
               link: /citing-scipy
             - text: Roadmap
-              link: https://scipy.github.io/devdocs/dev/roadmap.html
+              link: https://docs.scipy.org/doc/scipy/dev/roadmap.html
```

Comment:
This one could stay with dev docs as it should be updated as we go and it should be visible.